components/webgpu/wgpu_thread.rs

```diff
@@ -91,7 +93,7 @@ impl WGPU {
         let global = Arc::new(wgc::global::Global::new(
             "wgpu-core",
             InstanceDescriptor {
-                backends: wgt::Backends::PRIMARY,
+                backends: parse_backends_from_comma_list(&pref!(dom.webgpu.wgpu_backend)),
```

Comment:
I just remembered that we need to handle empty string on our own (as it is default value), so in case of empty strung we need to return `wgt::Backends::PRIMARY`.

Se relevant firefox code for reference (they complicate with cfgs for no reason): https://searchfox.org/mozilla-central/source/gfx/wgpu_bindings/src/server.rs#109